portal-api/api-tests/v1/gef/application.api-test.js

```diff
@@ -775,15 +777,12 @@ describe(baseUrl, () => {
       facilitiesToDelete = [new ObjectId(facility0._id), new ObjectId(facility1._id)];
     });
 
-    it('rejects requests that do not present a valid Authorization token', async () => {
-      const { status } = await as().remove(`${baseUrl}/${applicationToDeleteId}`);
-      expect(status).toEqual(401);
-    });
-
-    it('accepts requests that present a valid Authorization token with "maker" role', async () => {
-      const { status, body } = await as(aMaker).remove(`${baseUrl}/${applicationToDeleteId}`);
-      expect(status).toEqual(200);
-      expect(body).not.toEqual({ success: false, msg: "you don't have the right role" });
+    withRoleAuthorisationTests({
```

Comment:
Need to use other authorisation helper too

Reply:
Reminder to action